sdk/helper/keysutil/policy.go

```diff
@@ -1338,7 +1358,7 @@ func (p *Policy) VerifySignatureWithOptions(context, input []byte, sig string, o
 	}
 }
 
-func (p *Policy) Import(ctx context.Context, storage logical.Storage, key []byte, randReader io.Reader) error {
+func (p *Policy) Import(ctx context.Context, storage logical.Storage, key []byte, isPrivateKey bool, randReader io.Reader) error {
```

Comment:
If you want to make this easy on yourself, you can do:

```
func (p *Policy) Import(...) {
   p.ImportPublicOrPrivate(..., false, ...)
}

func (p *Policy) ImportPublicOrPrivate(....) ....
```

i.e., just sneak a helper in here and add the new parameter to that and call the helper from the existing method. Used a lot in places :-) 


Not a fan of my new name, you might find something better.